### ISSUE TITLE
Secondary sidebar visibility isn't persisted via File -> Open

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#173966]
1. Drag a view to the secondary sidebar such as search
2. Go to File -> Open and open a new folder
3. 🐛 Secondary sidebar is gone

This is annoying with views like explorer which update state with the workspace as it makes it difficult to see that the open occurred. If you open a folder from the recents list then it stays visible.

**Comment by sbatten** [comment]
Part visibility is workspace state

**Comment by lramos15** [comment]
Ah I see, the activity bar is by default visible but the other parts aren't. I see if you hide the activity bar or show the terminal they do the same. I guess it's whether or not we want to do something special for explorer then where what ever view contains the explorer should be revealed because it feels empty without it. I'm happy to close this as designed if you'd like

**Comment by sbatten** [comment]
I think if its to be fixed, it would come from an explorer contribution, not at the layout level.

**Comment by dinko-pehar** [comment]
I'm facing similar problem.

I started using secondary sidebar same as @lramos15 . I moved few items there, such as git pane, some extensions panes etc. Although these panes are kept in order when opening a new project, secondary sidebar is always closed.

@sbatten Is there any way to have it always open by default when opening new projects ? Do we have to open issue somewhere else for further discussion ? Thank you
